src/snippets/langsmith/set-workspace-secrets.mdx

```diff
@@ -0,0 +1,7 @@
+In the [LangSmith UI](https://smith.langchain.com) for your workspace, ensure that your OpenAI API key is set as a [workspace](/langsmith/administration-overview#workspace-secrets).
+
+1. Navigate to <Icon icon="gear" /> **Settings** and then move to the **Secrets** tab.
+1. On the **Workspace Secrets** page, select **Add secret** and enter the `OPENAI_API_KEY` and your API key as the **Value**.
```

Comment:
nit, "On the Workspace Secrets page" maybe redundant since above says "move to the Secrets tab"

Reply:
```suggestion
1. Select **Add secret**. Enter `OPENAI_API_KEY` as the **Key** and your API key as the **Value**.
```